src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -74,31 +71,29 @@ export const ScheduleItemNG: React.FC<ScheduleItemNGProps> = ({
     differenceInCalendarDays(eventEndTime(event), eventStartTime(event))
   );
   const isCurrentEventLive = isEventLive(event);
-  const roomUrlParam = getUrlParamFromString(event.room ?? "");
 
   const handleCopyEventLink = useCallback(
     (e?: React.MouseEvent<HTMLButtonElement>) => {
+      // @debt get rid of stopPropagation() in the project allowing a valid event bubbling
       e && e.stopPropagation();
```

Comment:
[OOS]
@sunny-viktoryia Need good example to always point it out while doing review